dashboard/src/repositories/postgres/dashboardConfig.ts

```diff
@@ -0,0 +1,71 @@
+import prisma from '@/lib/postgres';
+import {
+  DashboardConfig,
+  DashboardConfigSchema,
+  DashboardConfigCreate,
+  DashboardConfigUpdate,
+  withDashboardConfigCreateDefaults,
+} from '@/entities/dashboardConfig';
+
+export async function findDashboardConfigByDashboardId(dashboardId: string): Promise<DashboardConfig | null> {
+  try {
+    const cfg = await prisma.dashboardConfig.findUnique({ where: { dashboardId } });
+    if (!cfg) return null;
+    return DashboardConfigSchema.parse(cfg);
+  } catch (error) {
+    console.error('Error finding dashboard config by dashboardId:', error);
+    throw new Error('Failed to find dashboard config');
+  }
+}
+
+export async function createDashboardConfig(input: DashboardConfigCreate): Promise<DashboardConfig> {
+  try {
+    const { dashboardId, ...data } = input;
+    const cfg = await prisma.dashboardConfig.create({
+      data: { dashboardId, ...data },
+    });
```

Comment:
Why split them up, just to merge them together?
```suggestion
    const cfg = await prisma.dashboardConfig.create({
      data: input,
    });
```